### PR TITLE
Warn instead of error on missing entities.png

### PR DESCRIPTION
**Pull request by alexander-b**
Signed-off-by: Alexander Berntsen alexander@plaimi.net

### CODE REVIEW

**Comment by alexander-b** [comment]
Fixes #884.

**Comment by christopho** [comment]
If we decide to make it optional, there should be no error/warning at all.

**Comment by alexander-b** [comment]
Here is a commit that only complains when it's being used.

File: changelog.txt [hunk 1]
```diff
@@ -103,6 +103,7 @@ Changes that do not introduce incompatibilities:
 * Add methods sprite:get_size() and sprite:get_origin() (#823).
 * sprite:set_animation() now takes an optional callback parameter (#861).
 * Add a method surface:get_opacity() (#722).
+* Make "tileset.entities.png" files optional (#884).
```
Comment (christopho) [inline changelog.txt]: This should be in the "Data file format changes" section just below.

File: src/entities/Tileset.cpp [hunk 2]
```diff
@@ -194,6 +193,9 @@ const SurfacePtr& Tileset::get_tiles_image() {
  * \return the image containing the skin-dependent dynamic entities for this tileset
  */
 const SurfacePtr& Tileset::get_entities_image() {
+  if (entities_image == nullptr) {
+    Debug::error(std::string("Missing entities image for tileset '") + id + "': " + std::string("tilesets/") + id + ".dat" );
```
Comment (christopho) [inline src/entities/Tileset.cpp]: This will complain too many times I think. (Every time the image is requested)

**Comment by alexander-b** [comment]
I thought it might be useful to whine every time it would be accessed. I modified it now to only whine when set_tileset() is called. I also fixed moved changelog entry.

**Comment by christopho** [comment]
Closing this pull request. The entities.png file is now optional and there is no error or warning (see 6a02a21).

**Pull request closed without merging**